### PR TITLE
Replace expected output screenshot with code block

### LINKED ISSUE #241: Update the screenshot for the expected output in the main readme
The[ current screenshot](https://github.com/AlexsLemonade/scpca-downstream-analyses/blob/main/screenshots/expected_output_structure.png) of the expected output structure is a cutoff image, we should take a new screenshot that doesn't include the lines on the side.

### PR DESCRIPTION
**Pull request by cbethell**
**Issue Addressed**
Closes #241

**What is the purpose of these changes?**

This PR addresses #241 by replacing the slightly cutoff screenshot of the expect output with a code block of the expected output structure.

**What changes did you make?**

This PR removes the expected output screenshot from the `README` and repo, updates the expected output structure with a code block, and also updates the name of the `results_dir` to be `example_results` as we have it in the config file and repo. 

**Any comments, concerns, or questions important for reviewers**
- Does this PR seem to adequately address #241?

**Checklist**

Place an `x` in all boxes that you have completed.

- [ ] I have run the most recent version of the code
- [ ] If I am adding in reports or generating any plots, I have reviewed the necessary reports
- [x] I have added all necessary documentation (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -273,13 +273,17 @@ Changes will be pushed to the `main` branch once changes are ready for a new rel
 ## Expected output
 
 For each `SingleCellExperiment` and associated `library_id` used as input, the workflow will return two files: a processed `SingleCellExperiment` object containing normalized data and clustering results, and a summary html report detailing the filtering of low quality cells, dimensionality reduction, and clustering that was performed within the workflow.
-These files can be found in the `results_dir`, as defined in the `config.yaml` file.
-Within the `results_dir`, output files for each library will be nested within folders labeled with the provided `sample_id`.
+These files can be found in the `example_results`, as defined in the `config.yaml` file.
+Within the `example_results`, output files for each library will be nested within folders labeled with the provided `sample_id`.
```
Comment (sjspielman) [inline README.md:277]:
```suggestion
Within the `example_results` folder, output files for each library will be nested within folders labeled with the provided `sample_id`.
```

File: README.md [hunk 2]
```diff
@@ -273,13 +273,17 @@ Changes will be pushed to the `main` branch once changes are ready for a new rel
 ## Expected output
 
 For each `SingleCellExperiment` and associated `library_id` used as input, the workflow will return two files: a processed `SingleCellExperiment` object containing normalized data and clustering results, and a summary html report detailing the filtering of low quality cells, dimensionality reduction, and clustering that was performed within the workflow.
-These files can be found in the `results_dir`, as defined in the `config.yaml` file.
-Within the `results_dir`, output files for each library will be nested within folders labeled with the provided `sample_id`.
+These files can be found in the `example_results`, as defined in the `config.yaml` file.
```
Comment (sjspielman) [inline README.md:276]:
```suggestion
These files can be found in the `example_results` folder, as defined in the `config.yaml` file.
```

File: README.md [hunk 3]
```diff
@@ -273,13 +273,17 @@ Changes will be pushed to the `main` branch once changes are ready for a new rel
 ## Expected output
 
 For each `SingleCellExperiment` and associated `library_id` used as input, the workflow will return two files: a processed `SingleCellExperiment` object containing normalized data and clustering results, and a summary html report detailing the filtering of low quality cells, dimensionality reduction, and clustering that was performed within the workflow.
```
Comment (sjspielman) [inline README.md:275]: While we're here, let's capitalize HTML
```suggestion
For each `SingleCellExperiment` and associated `library_id` used as input, the workflow will return two files: a processed `SingleCellExperiment` object containing normalized data and clustering results, and a summary HTML report detailing the filtering of low quality cells, dimensionality reduction, and clustering that was performed within the workflow.
```

File: README.md [hunk 4]
```diff
@@ -273,13 +273,17 @@ Changes will be pushed to the `main` branch once changes are ready for a new rel
...
 Each output filename will be prefixed with the associated `library_id` and `filtering_method`.
 Below is an example of the nested file structure you can expect.
 
-![Expected output directory structure](./screenshots/expected_output_structure.png)
-
+```
+example_results
+└── sample_id
+	 ├── library_id_filtering_method_core_analysis_report.html
+	 └── library_id_filtering_method_processed_sce.rds
```
Comment (sjspielman) [inline README.md:285]: I'd suggest using `<>` here, as in: 

```suggestion
	 ├── <library_id>_<filtering_method>_core_analysis_report.html
	 └── <library_id>_<filtering_method>_processed_sce.rds
```

and then continue referring to these files with their "full names" as in next comments.

File: README.md [hunk 5]
```diff
@@ -273,13 +273,17 @@ Changes will be pushed to the `main` branch once changes are ready for a new rel
...
+```
 
 The `_processed_sce.rds` file is the [RDS file](https://rstudio-education.github.io/hopr/dataio.html#saving-r-files) that contains the final processed `SingleCellExperiment` object (which contains the filtered, normalized data and clustering results).
```
Comment (sjspielman) [inline README.md:288]:
```suggestion
The `<library_id>_<filtering_method>_processed_sce.rds` file is the [RDS file](https://rstudio-education.github.io/hopr/dataio.html#saving-r-files) that contains the final processed `SingleCellExperiment` object (which contains the filtered, normalized data and clustering results).
```

Comment (sjspielman) [reply README.md:288]: If you want to use this approach, also update accordingly on line 293 for the other file name.

**Review by sjspielman — APPROVED** [review]
Looks good! I don't need to see again, but others might disagree with my `<>` suggestion so let's see...

**Comment by cbethell** [comment]
I incorporated the `<>` because I do believe the example filenames are easier to read with them there 👍

**Pull request merged**